CMakeLists.txt

```diff
@@ -19,6 +19,21 @@ if (BIND_CONSOLE)
 	add_compile_definitions(BIND_CONSOLE)
 endif (BIND_CONSOLE)
 
+option(WINDOWED_MODE "Run windowed" OFF)
+if (WINDOWED_MODE)
+	add_compile_definitions(G2X_BUILD_OPTION_WINDOWED_MODE)
+endif ()
+
+option(SCREEN_WIDTH "Screen / window width" OFF)
```

Comment:
```option``` is just for ON/OFF values, so you cannot use it to pass integers. It won't work in the CMake GUI.
What you had before (with the default "max", which could be changed by "best") did work better.

Reply:
Thanks, fixed. Decided on empty string as default this time.